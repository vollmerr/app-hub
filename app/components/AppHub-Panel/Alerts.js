import React from 'react';

import Wrapper from './Wrapper';

class Alerts extends React.PureComponent {
  render() {
    return (
      <Wrapper>
        <h2>Alerts</h2>
        <hr />
        ...
      </Wrapper>
    );
  }
}

Alerts.propTypes = {

};

export default Alerts;
