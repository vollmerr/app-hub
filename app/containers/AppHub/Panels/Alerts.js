import React from 'react';

import Wrapper from './Wrapper';

class Alerts extends React.PureComponent {
  render() {
    return (
      <Wrapper>
        <h2>Alerts</h2>
        <hr />
        ...
      </Wrapper>
    );
  }
}


export default Alerts;
