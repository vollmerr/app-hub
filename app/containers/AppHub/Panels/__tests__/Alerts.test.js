import React from 'react';
import { shallow } from 'enzyme';

import Alerts from '../Alerts';


describe('<Alerts />', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Alerts />);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render a h2 header', () => {
    expect(wrapper.find('h2').length).toEqual(1);
  });

  xit('should render all the alerts', () => {
    // expect(wrapper.find(AlertsItem?).length).toEqual(alerts.length?);
  });
});
