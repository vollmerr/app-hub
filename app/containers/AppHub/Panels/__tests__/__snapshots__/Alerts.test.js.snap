// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Alerts /> should render correctly 1`] = `
<Wrapper__Wrapper>
  <h2>
    Alerts
  </h2>
  <hr />
  ...
</Wrapper__Wrapper>
`;
